src/Aspire.ProjectTemplates/templates/aspire-starter/9.2/Aspire-StarterApplication.1.Tests/Aspire-StarterApplication.1.Tests.csproj

```diff
@@ -21,8 +25,12 @@
     <PackageReference Include="NUnit" Version="4.2.2" Condition=" $(TestFramework) == 'NUnit' " />
     <PackageReference Include="NUnit.Analyzers" Version="4.2.0" Condition=" $(TestFramework) == 'NUnit' " />
     <PackageReference Include="NUnit3TestAdapter" Version="4.5.0" Condition=" $(TestFramework) == 'NUnit' " />
-    <PackageReference Include="xunit" Version="2.9.2" Condition=" $(TestFramework) == 'xUnit.net' " />
-    <PackageReference Include="xunit.runner.visualstudio" Version="2.8.2" Condition=" $(TestFramework) == 'xUnit.net' " />
+    <PackageReference Include="xunit" Version="2.9.2"  Condition=" $(XUnitVersion) == 'v2' "/>
+    <PackageReference Include="xunit.runner.visualstudio" Version="3.0.2" Condition=" $(TestFramework) == 'xUnit.net' " />
```

Comment:
guidance is to keep this runner pkg reference even when using xunit v3, for backcompat.
https://xunit.net/docs/getting-started/v3/microsoft-testing-platform

the v3 runner is built in.